PVSSpringBoot/src/main/java/com/example/PVSSpringBoot/OAuth2/AOuth2Service.java

```diff
@@ -0,0 +1,56 @@
+package com.example.PVSSpringBoot.OAuth2;
+import com.example.PVSSpringBoot.Entities.User;
+import com.example.PVSSpringBoot.repositories.UsersRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import java.sql.Date;
+import java.util.Optional;
+
+public class AOuth2Service {
+    UsersRepo usersRepo;
+    @Autowired
+    public AOuth2Service(UsersRepo uusersRepo) {
+        this.usersRepo = uusersRepo;
+    }
+public String BuildOAuthSignIn(String OAuth){
+    String[] arrOfStr = OAuth.split("Credentials=");
+    String[] arrOfStr2 = arrOfStr[0].split("email=");
+    String[] arrOfStr3 = arrOfStr[0].split("name=");
+    arrOfStr3 = arrOfStr3[1].split(",");
+    String[] arrOfStr4 = arrOfStr[0].split("family_name=");
```

Comment:
We don't need arrOfStr3 or arrOfStr4 so we should remove them 
We just need the email of the user